packages-next/keystone/package.json

```diff
@@ -4,6 +4,17 @@
   "license": "MIT",
   "main": "dist/keystone.cjs.js",
   "module": "dist/keystone.esm.js",
+  "files": [
+    "session",
+    "bin",
+    "artifacts",
+    "___internal-do-not-use-will-break-in-patch",
+    "migrations",
+    "next",
+    "schema",
+    "scripts",
+    "session"
+  ],
```

Comment:
This is a thing for future PRs that I forgot I did now and don't really feel like reverting so that we don't publish `src` to npm because it will contain tests soon and they could get large.